CHANGELOG.md

```diff
@@ -20,7 +20,7 @@ BEGIN_UNRELEASED_TEMPLATE
 
 ### New
 
-* TBD
+* Set remote_download_outputs to toplevel as required to download needed outputs
```

Comment:
Please link to this PR.

Also, please put it in the `Adjusted` section and use this for the title:
```
We now set `--remote_download_outputs=toplevel` on the `rules_xcodeproj` config
```